server.js

```diff
@@ -1 +0,0 @@
-require("./server/main");
```

Comment:
server files are now built into `dist/server` and `npm serve` command points to `dist/server/main.js`